tests/service/Symbols.fs

```diff
@@ -751,3 +752,17 @@ type Foo() =
             and set (a: int) (b: float) = ()
 """
             (5, 14, "    member _.X", "X")
+            
+module AnonymousRecord =
+    [<Test>]
+    let ``Anonymous record copy-and-update symbols`` () =
+        let _, checkResults = getParseAndCheckResults """
+let f (r: {| A: int; C: int |}) =
+    { r with A = 1; B = 2; C = 3 }
+"""
+        assertHasSymbolUsages [ "A"; "C" ] checkResults
```

Comment:
This doesn't actually check the usages of `A`, and `C`. Since they are declared in the type annotation, they will make this check pass regardless of whether the actual usages are reported below.

Reply:
Ok. let me check if there is any example of how to do that. I'm open to suggestions

Reply:
Updated

Reply:
Previously it tested normal record syntax, and now these tests check anon records syntax instead. Please don't remove tests for the normal syntax.

Reply:
Done

Reply:
@edgarfgp Thanks!